lib/generators/solidus_paypal_commerce_platform/install/install_generator.rb

```diff
@@ -5,14 +5,36 @@ module Generators
     class InstallGenerator < Rails::Generators::Base
       class_option :migrate, type: :boolean, default: true
       class_option :backend, type: :boolean, default: true
-      class_option :starter_frontend, type: :boolean, default: true
+      class_option :frontend, type: :string, default: 'starter'
+
+      # Deprecated
+      class_option :starter_frontend, type: :boolean
 
       # This is only used to run all-specs during development and CI,  regular installation limits
       # installed specs to frontend, which are the ones related to code copied to the target application.
       class_option :specs, type: :string, enum: %w[all frontend], default: 'frontend', hide: true
 
       source_root File.expand_path('templates', __dir__)
 
+      def normalize_components_options
+        @components = {
+          backend: options[:backend],
+        }
+
+        if options[:starter_frontend].present?
+          message = <<~TEXT
+            --starter-frontend option is deprecated. Please use --frontend=starter/classic instead.
+          TEXT
+          say_status :deprecation, set_color(message.gsub("\n", ' '), :yellow), :yellow
+
+          if options[:starter_frontend]
+            @components[:starter_frontend] = true
+          end
+        end
+
+        @components[:starter_frontend] = true if options[:frontend] == 'starter'
+      end
```

Comment:
Do we actually have legacy users that were able to use this flag?

If it just lived on master (or on a beta) for some time I'd say we can skip the deprecation, thoughts?

Reply:
You are right. I was assuming we were using the flag here: https://github.com/solidusio/solidus/blob/843a9a166845facfc92afe65423de2e7a7e7d7ca/core/lib/generators/solidus/install/app_templates/payment_method/paypal.rb#L10 but the default is used.